vite.config.ts

```diff
@@ -22,4 +22,9 @@ export default defineConfig({
     },
     open: true
   },
+  resolve: {
+    alias: {
+      src: '/src'
```

Comment:
```suggestion
      src: './src'
```

This alias looks strange to me, can you describe this in more detail?

Reply:
I spent some time looking over the docs to try and understand the normal vite pattern, and `./src` is the common usage. However, it doesn't work for us - vite can't seem to resolve that. It does work with `/src`. If there's a way to make `./src` work, I'm open to whatever that is.